products/appengine/api.yaml

```diff
@@ -440,3 +440,39 @@ objects:
             name: 'shell'
             description: |
               The format should be a shell command that can be fed to bash -c.
+  - !ruby/object:Api::Resource
+    name: 'UrlDispatchRule'
+    description: |
+      Rules to match an HTTP request and dispatch that request to a service.
+    base_url: ''
+    create_url: 'apps/{{project}}?updateMask=dispatch_rules'
+    create_verb: :PATCH
+    delete_url: 'apps/{{project}}?updateMask=dispatch_rules'
+    delete_verb: :PATCH
+    update_url: 'apps/{{project}}?updateMask=dispatch_rules'
+    update_verb: :PATCH
+    update_mask: true
```

Comment:
I suspect that this will end up adding an extra update mask we don't need. Do you mind removing it?

Reply:
Removing delete and update, this was added as per recommendation by Emily.

Reply:
Without a `base_url` set, we won't have a value for the URI in delete and update. Can you add those back, and remove this line instead?

Reply:
I added those back, not sure which line you want me to delete.